example-android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri May 19 11:15:00 EEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
You need to run `./gradlew wrapper --gradle-version 8.1.1` to upgrade. Just bumping `gradle-wrapper.properties` does not update the `gradlew` scripts or jar. Those need to be checked in as well.

Reply:
Thank you. Fixed